packages/core/src/utils/breakpoints.ts

```diff
@@ -110,19 +114,20 @@ export const getValueForBreakpoint = (
     // Assume that the values are sorted by media query to apply the cascading CSS logic
     for (let index = activeBreakpointIndex; index >= 0; index--) {
       const breakpointId = breakpoints[index]?.id;
-      if (valuesByBreakpoint[breakpointId]) {
+      if (isValidBreakpointValue(valuesByBreakpoint[breakpointId])) {
         // If the value is defined, we use it and stop the breakpoints cascade
         return eventuallyResolveDesignTokens(valuesByBreakpoint[breakpointId]);
       }
     }
     // If no breakpoint matched, we search and apply the fallback breakpoint
     const fallbackBreakpointIndex = getFallbackBreakpointIndex(breakpoints);
     const fallbackBreakpointId = breakpoints[fallbackBreakpointIndex]?.id;
-    if (valuesByBreakpoint[fallbackBreakpointId]) {
+    if (isValidBreakpointValue(valuesByBreakpoint[fallbackBreakpointId])) {
       return eventuallyResolveDesignTokens(valuesByBreakpoint[fallbackBreakpointId]);
     }
   } else {
     // Old design properties did not support breakpoints, keep for backward compatibility
     return valuesByBreakpoint;
   }
+  return undefined;
```

Comment:
You probably don't need to explicitly return undefined here, it would return `undefined` anyway if it reached this point